[HB-5703] Document New Chartboost Mediation Adapter Window Capabilities (#122)
ChartBoost/chartboost-mediation-unity-sdk: com.chartboost.mediation/Documentation/setup/ad-adapters.md

Each Chartboost Mediation Unity SDK sample contains a .xml file with dependencies for the adapter available platforms (Android/iOS). In order for such dependencies to be included into the final build the [Google's External Dependency Manager](https://github.com/googlesamples/unity-jar-resolver) is needed.
The `EDM` makes sure to download and package any dependencies with your build.

## Chartboost Mediation Adapters Window

> Since Chartboost Mediation 4.X Adapters are no longer released at the cadence as the Chartboost Mediation SDK, this means is possible to receive adapter updates in between SDK updates. 

Reviewer: ```suggestion
Since Chartboost Mediation 4.X Adapters are no longer released at the cadence as the Chartboost Mediation SDK, it is now possible to receive adapter updates in between SDK releases. 
```

> Until Chartboost Mediation Unity SDK 4.1.0, adapters were addded through the UPM Samples capability; however, this limited the ability to provide adapter updates in between SDK updates. As such, from Chartboost Mediation Unity SDK 4.2.0, we have created a brand new Editor Window, that will allow users to fetch Ad Adapter updates on demand. *see screenshot below*

Reviewer: ```suggestion
As of Chartboost Mediation Unity SDK 4.1.0, adapters were added through the UPM Samples capability; however, this limited the ability to provide adapter updates in between SDK releases. As such, from Chartboost Mediation Unity SDK 4.2.0, we have created a brand new Editor Window. This will allow users to fetch Ad Adapter updates on demand. *see screenshot below*
```
